### PR TITLE
Removed outdated links

### PR DESCRIPTION
**Pull request by rmdash**
Added coingecko link.

Still need clarity on TG links.

Preview build: https://dash-docs--268.org.readthedocs.build/en/268/

### CODE REVIEW

**Review by strophy — APPROVED** [review]

File: docs/user/introduction/information.rst [hunk 1]
```diff
@@ -104,22 +93,17 @@ Facebook
...
 - **Thailand:** https://www.facebook.com/groups/dashembassythailand
-- **Venezuela:** https://www.facebook.com/groups/DarkcoinVenezuela
 - **Vietnam:** https://www.facebook.com/dashvietnam
 
 
 Twitter
 -------
 
 - **Dash Official Account:** https://twitter.com/dashpay
-- **Ryan Taylor, CEO of Dash Core Group:** https://twitter.com/RTaylor05
```
Comment (thephez) [inline docs/user/introduction/information.rst:115]: Should we put Sam's here? https://twitter.com/SamWestrich/

File: docs/user/introduction/information.rst [hunk 2]
```diff
@@ -30,27 +29,17 @@ Community sites
...
 - https://dashdevs.org
-- https://app.dashnexus.org
-- https://www.dashwatch.org
 - https://www.dashcentral.org
 - https://www.dashninja.pl
 - https://dashnews.org
-- https://dashthailand.org
-- https://dashnigeria.com/
-- https://www.dashnation.com
 - https://dashvotetracker.com
```
Comment (thephez) [inline docs/user/introduction/information.rst:35]: @strophy This page is super out of date. Should we keep for historical info?
```suggestion
- https://dashvotetracker.com
```

File: docs/user/introduction/information.rst [hunk 3]
```diff
@@ -104,22 +93,17 @@ Facebook
...
-- **Venezuela:** https://www.facebook.com/groups/DarkcoinVenezuela
 - **Vietnam:** https://www.facebook.com/dashvietnam
 
 
 Twitter
 -------
 
 - **Dash Official Account:** https://twitter.com/dashpay
-- **Ryan Taylor, CEO of Dash Core Group:** https://twitter.com/RTaylor05
 - **Dash Force News:** https://twitter.com/DashForceNews
```
Comment (thephez) [inline docs/user/introduction/information.rst:103]: No activity in 3 yrs
```suggestion
```

File: docs/user/introduction/information.rst [hunk 4]
```diff
@@ -104,22 +93,17 @@ Facebook
...
 - **Joël Valenzuela, Chief Editor, Dash Force News:** https://twitter.com/TheDesertLynx
-- **Mark Mason, Director of Media & PR, Dash Force News:** https://twitter.com/StayDashy
-- **Amanda B. Johnson, Dash Superfan and DAO-funded contractor:** https://twitter.com/AmandaB_Johnson
 - **Tao of Satoshi, Dash Nation Founder and DAO-funded contractor:** `https://twitter.com/Dash_Nation <https://twitter.com/Dash_Nation>`__
```
Comment (thephez) [inline docs/user/introduction/information.rst:105]: Nothing recent about Dash on here
```suggestion
```

**Review by thephez — COMMENTED** [review]
A couple changes

File: docs/user/introduction/information.rst (hunk 1 above)
Comment (rmdash) [reply docs/user/introduction/information.rst:115]: added and pushed

**Pull request merged**